lib/saito/ui/game-hexgrid/game-hexgrid.template.js

```diff
@@ -18,11 +16,11 @@ module.exports = GameHexGridTemplate = (height, width, mode=[]) => {
   html += "\n";
 
   // rendering problem fix -- empty top row
-  if ((height+1)%2 == 0) {
```

Comment:
Ingeneral you always want `===`, see i.e.: https://howtodoinjava.com/javascript/javascript-equality-vs-identity-operators/